test/e2e/leaderelection.go

```diff
@@ -76,17 +76,19 @@ var _ = Describe("Leader Elector", func() {
 })
 
 func spawnControllerManagerProcess(tl common.TestLogger, kubeConfigPath, namespace string) (*exec.Cmd, io.ReadCloser, error) {
-	// Get the directory of the current executable
-	_, filename, _, _ := runtime.Caller(0)
-	managedPath := filepath.Dir(filename)
-	confFile, err := filepath.Abs(fmt.Sprintf("%s/../../config/kubefedconfig.yaml", managedPath))
+	kubefedconfigAsset := common.MustAsset("config/kubefedconfig.yaml")
+	tmpFile, err := ioutil.TempFile(os.TempDir(), "kubefedconfig.yaml")
```

Comment:
(No action required) There is no requirement for the file to be named `kubefedconfig.yaml`, it could instead be truly random.  Also, it's not necessary to pass `os.TempDir()` to `ioutil.TempFile` - passing an empty string has the same result.